packages/xo-server/src/xapi-object-to-xo.mjs

```diff
@@ -7,6 +9,8 @@ import { extractProperty, forEach, isEmpty, mapFilter, parseXml } from './utils.
 import { getVmDomainType, isHostRunning, isVmRunning, parseDateTime } from './xapi/index.mjs'
 import { useUpdateSystem } from './xapi/utils.mjs'
 
+const MAINTAINED_VERSION = JSON.parse(fs.readFileSync(new URL('../maintainedVersion.json', import.meta.url)))
```

Comment:
Why do you need all this? 
 You can inline it.

Reply:
`MAINTAINED_HOST_VERSION`?

Reply:
I need this to read the content of `maintainedVersion.json`.